coreprofile/11/_index.md

```diff
@@ -21,18 +25,20 @@ The Jakarta EE Core Profile defines a profile of the Jakarta EE platform specifi
 
 * [Jakarta EE Core Profile 11 Release Record](https://projects.eclipse.org/projects/ee4j.jakartaee-platform/releases/core-profile-11)
     * [Jakarta EE Platform 11 Release Plan](https://jakartaee.github.io/platform/jakartaee11/JakartaEE11ReleasePlan)
-* [Jakarta EE 11 M4 core profile Specification Document](./jakarta-coreprofile-spec-11.0-M4.pdf) (PDF)
-* [Jakarta EE 11 M4 core profile Specification Document](./jakarta-coreprofile-spec-11.0-M4.html) (HTML)
-* [Jakarta Core Profile 11 M4 Javadoc](./apidocs)
+* [Jakarta EE 11 core profile Specification Document](./jakarta-coreprofile-spec-11.0.pdf) (PDF)
+* [Jakarta EE 11 core profile Specification Document](./jakarta-coreprofile-spec-11.0.html) (HTML)
```

Comment:
I think it would be better if "core profile" here was capitalised. This would better align it with the usage in the other bullet points
```suggestion
* [Jakarta EE 11 Core Profile Specification Document](./jakarta-coreprofile-spec-11.0.pdf) (PDF)
* [Jakarta EE 11 Core Profile Specification Document](./jakarta-coreprofile-spec-11.0.html) (HTML)
```

Reply:
@starksm64 please commit this suggestion to your PR